spec/models/manageiq/providers/vmware/cloud_manager_spec.rb

```diff
@@ -121,5 +121,15 @@
         @ems.vm_create_snapshot(vm, snapshot_options)
       end
     end
+
+    context ".vm_remove_snapshot" do
+      let(:connection) { double("connection", :post_remove_all_snapshots => "post_remove_all_snapshots") }
```

Comment:
```
let(:connection) { double("connection") }
```

I don't think you really need to specify post_remove_all_snapshots attribute because of the `expect(connection).to receive(:post_remove_all_snapshots)` below.